packages/loader/container-utils/src/test/error.spec.ts

```diff
@@ -90,7 +90,7 @@ describe("Errors", () => {
             assert(coercedError.errorType === ContainerErrorType.dataProcessingError);
             assert(coercedError.getTelemetryProperties().dataProcessingError === 1);
             assert(coercedError.getTelemetryProperties().dataProcessingCodepath === "someCodepath");
-            assert(coercedError.getTelemetryProperties().untrustedOrigin === 1);
+            assert(coercedError.getTelemetryProperties().untrustedOrigin === undefined);
```

Comment:
Also add a test case for a normalized LoggingError?

Reply:
Sounds good. I have added the changes in commit #a1d49538.